src/config/paths.js

```diff
@@ -14,6 +14,8 @@ export const EXECUTIONS_PATH = '/executions';
 export const VISUALIZATIONS_PATH = '/visualizations';
 export const SCHEMAS_PATH = '/schemas';
 export const SCHEMA_PATH = '/schema/:id';
+export const PIPELINES_PATH = '/pipelines';
+export const CREATE_PIPELINE_PATH = '/create-pipeline';
```

Comment:
Since we agreed on `addPipeline` maybe change this accordingly.

Reply:
So do I need to change the name for `CreatePipeline.js`  too ?

Reply:
Ideally yes.